compose/snippets/src/main/java/com/example/compose/snippets/text/TextFieldMigrationSnippets.kt

```diff
@@ -0,0 +1,331 @@
+package com.example.compose.snippets.text
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.text.input.InputTransformation
+import androidx.compose.foundation.text.input.OutputTransformation
+import androidx.compose.foundation.text.input.delete
+import androidx.compose.foundation.text.input.insert
+import androidx.compose.foundation.text.input.TextFieldLineLimits
+import androidx.compose.foundation.text.input.TextFieldState
+import androidx.compose.foundation.text.input.maxLength
+import androidx.compose.foundation.text.input.rememberTextFieldState
+import androidx.compose.foundation.text.input.setTextAndPlaceCursorAtEnd
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.SecureTextField
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.Text
+//noinspection UsingMaterialAndMaterial3Libraries
```

Comment:
Unfortunately the latest Material3 stable release does not have the new TextField. It's still in alpha track.